### PR TITLE
feat: scheduling speed enhancement / bug fix

### PR DESCRIPTION
**Pull request by dodamih**

### CODE REVIEW

File: zetta_utils/geometry/bbox.py [hunk 1]
```diff
@@ -443,6 +443,14 @@ def aligned(self, other: BBox3D) -> tuple[bool, ...]:  # pragma: no cover
         x_start, x_stop, y_start, y_stop, z_start, z_stop order."""
         return tuple(s[i] == o[i] for s, o in zip(self.bounds, other.bounds) for i in range(2))
 
+    def contained_in(self: BBox3D, other: BBox3D) -> bool:  # pragma: no cover
```
Comment (supersergiy) [inline zetta_utils/geometry/bbox.py:446]: This needs a test

File: zetta_utils/mazepa_layer_processing/common/volumetric_apply_flow.py [hunk 2]
```diff
@@ -347,20 +348,25 @@ def _get_temp_dst(
...
         op_kwargs: P.kwargs,
     ) -> List[mazepa.tasks.Task[R_co]]:
-        tasks = [
-            self.op.make_task(
-                idx=idx_chunk,
-                dst=dst,
-                **op_kwargs,
+        with multiprocessing.Pool() as pool_obj:
+            tasks = pool_obj.map(
+                self._make_task,
```
Comment (supersergiy) [inline zetta_utils/mazepa_layer_processing/common/volumetric_apply_flow.py:367]: Why is the `self._make_task` necessary here? Can we just map `self.op.make_task`?

**Review by supersergiy — CHANGES_REQUESTED** [review]

File: zetta_utils/mazepa_layer_processing/common/volumetric_apply_flow.py (hunk 2 above)
Comment (dodamih) [reply zetta_utils/mazepa_layer_processing/common/volumetric_apply_flow.py:367]: Cannot because parallel map requires a single argument.

Comment (supersergiy) [reply zetta_utils/mazepa_layer_processing/common/volumetric_apply_flow.py:367]: Gotcha. In that case it's fine for the helper to be not typed tightly

**Review by supersergiy — APPROVED** [review]
Looks great!

**Pull request merged**